app/containers/User/Invite.js

```diff
@@ -40,9 +40,10 @@ export const Invite = React.createClass({
 
 		// Facebook
 		const facebookTitle = encodeURI('Fifty Nifty');
+		const facebookAppID = '243568346101604';
```

Comment:
Have you created a Facebook App for the project? We already have one. We've been using it for the card validator. Check App.js for `fb:app_id`. Let's use that one since it was created first. We can sync up on any settings you've added to make sure those are added. I don't want to merge until we clear this up, so we don't split facebook traffic across two different apps.